spec/system/employee/earned_points_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'Employee earned points', type: :system do
+  let(:employee) { create(:employee) }
+
+  context 'when emplyee is signed in' do
+    it 'Navbar displays earned points for employee' do
+      sign_in employee
+      visit root_path
+      expect(page).to have_content 'Points: 0'
+      kudo = create(:kudo, reciever: employee)
```

Comment:
mixing user actions with FActoryBot actions looks a bit weird - never seen anything like it :) I would rather consider  putting the `create` methods in the `before` block